pynitrokey/cli/start.py

```diff
@@ -93,14 +95,81 @@ def rng(count, raw, quiet):
                 print(challenge.hex())
 
 
+class CardRemovedGPGAgentException(RuntimeWarning):
+    pass
+
+
+def gpg_agent_set_identity(identity: int):
+    from pexpect import run
+
+    cmd = f"gpg-connect-agent 'SCD APDU 00 85 00 0{identity}' /bye"
```

Comment:
It could make sense to run [`SERIALNO`](https://www.gnupg.org/documentation/manuals/gnupg/Scdaemon-SERIALNO.html#Scdaemon-SERIALNO) before this one:

> This command should be used to check for the presence of a card. It is special in that it can be used to reset the card. Most other commands will return an error when a card change has been detected and the use of this function is therefore required.